### DISCUSSION TITLE
IOPS values ​​sometimes collected too high

### DISCUSSION THREAD

**Discussion by wooyoungAhn** [NetApp/harvest#3321] [General]
Hello

Sometimes, Harvest reports a total of 1 billion IOPS. However, this value is not actually used by the ONTAP Cluster.

Although I cannot provide a screenshot from the client at the moment, the reported value appears unreasonably high for a very short period of time, similar to the graph below.
![image](https://github.com/user-attachments/assets/ee9a6603-c644-4ace-b0c6-9c300fefe732)
Currently, the customer has over 120 cluster connected to a single Harvest system.
Could having too many pollers connected cause Harvest to report incorrect values?

**Comment by rahulguptajss** [comment]
@wooyoungAhn Could you share Harvest version?
We have fixed similiar issues in past.
https://github.com/NetApp/harvest/issues/2544
https://github.com/NetApp/harvest/issues/2769
https://github.com/NetApp/harvest/issues/2605

**Reply by wooyoungAhn** [reply]
Harvest version 24.11.0

**Reply by rahulguptajss** [reply]
Could you please upload the poller logs to the following link?
[https://upload.nabox.org/hoty-vege-jyro](https://upload.nabox.org/hoty-vege-jyro)

Do you observe spikes only in these specific metrics? If so, how frequently do these spikes occur?
Could you also share the name of the metric shown in the screenshot you shared above?

**Reply by rahulguptajss** [reply]
@wooyoungAhn If this metric name is `node_volume_total_ops`, then there is a fix which went live in release [24.11.1](https://github.com/NetApp/harvest/releases/tag/v24.11.1) a few minutes ago. The issue was fixed via [this pull request](https://github.com/NetApp/harvest/pull/3324).

Could you ask the customer to upgrade to this build and see if it resolves the issue?

**Reply by rahulguptajss** [reply]
@wooyoungAhn Any updates on this? Thanks.

**Reply by wooyoungAhn** [reply]
sorry i'm late
I see similar symptoms in the ```qtree_nfs_ops metric```
Is this also subject to modification?

**Reply by rahulguptajss** [reply]
@wooyoungAhn This is not expected for `qtree_nfs_ops`. 
Could you please upload the relevant poller logs to the following link?
https://upload.nabox.org/tuny-byto-repo

**Reply by wooyoungAhn** [reply]
I will upload it after the version update. Thank you

**Reply by rahulguptajss** [reply]
Please upload logs once you notice the issue with new version.
